src/components/Fields/Contract/Contract.tsx

```diff
@@ -19,6 +19,10 @@ export const Contract = () => {
   }, [DODAAC, resetField]);
 
   const options: ContractOptions = [];
+  options.push({
```

Comment:
Maybe set a condition on pushing this. Condition being they must have a DODAAC. Otherwise, if they skip over the DODAAC, and select this, and a Contractor and Contract POC, and then select the DODAAC, it wipes away their other selections